components/Header.tsx

```diff
@@ -28,7 +28,7 @@ const Header = () => (
         <a className="text-lg text-gray-500 pointer-events-auto" >Why?</a>
       </Link>
     </div>
-    <Search />
+    {/* <Search /> */}
```

Comment:
temporarily commented this out as it was creating a full-height empty column on right side of screen, preventing entry page from taking full width (maybe solve-able with a z index, but would argue entry page doesn't need search visible anyways!)